run_test.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import argparse
+import os
+import re
+import subprocess
+import sys
+import time
+
+
+def quiet_call(args):
+    subprocess.check_call(args, stdout=subprocess.DEVNULL,
+                          stderr=subprocess.DEVNULL)
+
+
+class AdbConnection:
```

Comment:
I wonder if a similar adb harness exists in m-c somewhere (part of mach?) or in a third-party library.